output.txt

```diff
@@ -0,0 +1,28 @@
+(cd provider && go build -p 2 -o /Users/flo/development/pulumi-aws/bin/pulumi-tfgen-aws -ldflags "-X github.com/pulumi/pulumi-aws/provider/v6/pkg/version.Version=6.0.0-alpha.0+dev -X github.com/hashicorp/terraform-provider-aws/version.ProviderVersion=6.0.0-alpha.0+dev " github.com/pulumi/pulumi-aws/provider/v6/cmd/pulumi-tfgen-aws)
```

Comment:
This file should not be checked in possibly?

Reply:
Argh, I hate that upgrade-provider picks up random files.